app/src/main/kotlin/com/intive/picover/common/validator/TextValidator.kt

```diff
@@ -8,24 +8,23 @@ import com.intive.picover.common.validator.ValidationStatus.ValidText
 class TextValidator private constructor(
 	private val allowEmpty: Boolean,
 	private val allowBlank: Boolean,
-	private val maxLength: Int?,
 ) {
 
-	fun validatingText(text: String) =
-		when {
-			!allowEmpty && text.isEmpty() -> EmptyText
-			!allowBlank && text.isBlank() -> BlankText
-			maxLength != null && text.length > maxLength -> TooLongText
-			else -> ValidText
-		}
+	fun validatingText(
+		text: String,
+		isLongText: Boolean = false,
```

Comment:
Why it is passed each time to `validatingText` instead of storing it in `val` like `allowEmpty` and `allowBlank`? We could have two instances of `TextValidator` – one for short text and one for long text.

Reply:
Updated